axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -1099,6 +1099,95 @@
           "address": "0x9186233C327329408a9034F22407244421cADbEF"
         }
       }
+    },
+    "hyperliquid": {
+      "name": "Hyperliquid",
+      "axelarId": "hyperliquid",
+      "networkType": "testnet",
+      "chainId": 998,
+      "rpc": "https://rpc.hyperliquid-testnet.xyz/evm",
+      "tokenSymbol": "HYPE",
+      "confirmations": 1,
+      "finality": "finalized",
+      "decimals": 18,
+      "approxFinalityWaitTime": 1,
+      "chainType": "evm",
+      "explorer": {
+        "name": "Hyperliquid-testnet Explorer",
+        "url": "https://app.hyperliquid-testnet.xyz/explorer"
+      },
+      "contracts": {
+        "AxelarGateway": {
+          "deployer": "0x640fF4a6D70AD92fc064633586D0A51FA8D89Ba5",
+          "operator": "0x640fF4a6D70AD92fc064633586D0A51FA8D89Ba5",
+          "proxyDeploymentArgs": [
+            "0x23FcB411f022F887ce811eFD5C5d590bdAA1EEA3",
+            "0x640fF4a6D70AD92fc064633586D0A51FA8D89Ba5",
+            "0x000000000000000000000000640ff4a6d70ad92fc064633586d0a51fa8d89ba50000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000006993c6000000000000000000000000000000000000000000000000000000000000000300000000000000000000000051380cbf0777990e197a3e498ffafd26143e35f8000000000000000000000000000000000000000000000000000000000000000100000000000000000000000061373485594010371dfbf3a8f8bd1736bfda7c090000000000000000000000000000000000000000000000000000000000000001000000000000000000000000c70aa87b38e6ab3c0df6e56338f96a5c00e653080000000000000000000000000000000000000000000000000000000000000001"
+          ],
+          "initialVerifierSetId": "74acaec799e88ebd2c923682b8f0416184e72a7e4e878517d22fda44f30bf0a2",
+          "address": "0x8bCF77225946Fe5c44013CC241076b3995CA1018",
+          "implementation": "0x23FcB411f022F887ce811eFD5C5d590bdAA1EEA3",
+          "implementationCodehash": "0xddac21f07c3549f6971319b0b65bbaedcf4215f52a96c9dc368af419e35ee860",
+          "deploymentMethod": "create",
+          "previousSignersRetention": 15,
+          "domainSeparator": "0x2d921bc929b5f1f775bfd6984216cf844ff00e80e68f95f7c8eb633d9a7a45e5",
+          "minimumRotationDelay": 0,
+          "connectionType": "amplifier",
+          "owner": "0x640fF4a6D70AD92fc064633586D0A51FA8D89Ba5"
+        },
+        "ConstAddressDeployer": {
```

Comment:
if you see other chains, we reuse the same create2/create3 factories for devnet. doesn't matter for test deployments. useful for official deployments for consistency